glooshot/docs/installation/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Installation
+menuTitle: Installing
+weight: 2
+---
+
+
+
+{{% children description="true" %}}
```

Comment:
Now that I think about... Should we just have a single level for Installation? Ie, no nested "quick start" and just put it on the top level? We add more clicking when not needed at the moment to get to installation.

Reply:
Good point, I did the same for the tutorial